### PR TITLE
Create routes for edu-benefits

### LINKED ISSUE #2823: Create routes
Routes will be:
1. /introduction
2. /benefits-eligibility/benefits-selection
3. /military-history/military-service
4. /military-history/additional-information
4. /military-history/ROTC-history
5. /military-history/benefits-history
6. /education-history/education-information
7. /education-history/additional-information
8. /employment-history/employment-information
9. /school-selection/school-information
10. /veteran-information/personal-information
11. /veteran-information/address
12. /veteran-information/contact-information
13. /veteran-information/secondary-contact
14. /veteran-information/dependent-information
15. /veteran-information/direct-deposit
16. /review-and-submit

### PR DESCRIPTION
**Pull request by jbalboni**
Closes #2823.

This creates routes for each of the form sections for edu benefits. It also moves the veteran information section to the beginning of the Nav and adds some sections that were missing.

### CODE REVIEW

File: src/js/edu-benefits/containers/EduBenefitsApp.jsx [hunk 1]
```diff
@@ -5,14 +5,14 @@ import { connect } from 'react-redux';
...
 
+import PerfPanel from '../components/debug/PerfPanel';
+import RoutesDropdown from '../components/debug/RoutesDropdown';
+
+
 class EduBenefitsApp extends React.Component {
   render() {
     let devPanel = undefined;
     if (__BUILDTYPE__ === 'development') {
-      // using require here allows this code to be removed in prod
```
Comment (jbalboni) [inline src/js/edu-benefits/containers/EduBenefitsApp.jsx]: This is unnecessary now, but forgot to remove it.

File: src/js/hca/components/HealthCareApp.jsx [hunk 2]
```diff
@@ -12,8 +12,6 @@ import { ensureFieldsInitialized, updateCompletedStatus, updateSubmissionStatus,
 import { veteranToApplication } from '../../common/model/veteran';
 import * as validations from '../../common/utils/validations';
 
-// TODO(awong): Find some way to remove code when in production. It might require System.import()
```
Comment (jbalboni) [inline src/js/hca/components/HealthCareApp.jsx]: This is handled in Webpack now, so removing the comment.

**Comment by jbalboni** [comment]
@akainic Want to review this?

File: src/js/hca/components/HealthCareApp.jsx (hunk 2 above)
Comment (akainic) [inline src/js/hca/components/HealthCareApp.jsx]: Awesome, thank you!

**Comment by akainic** [comment]
LGTM!

**Pull request merged**